LICENSE

```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    {one line to give the program's name and a brief idea of what it does.}
-    Copyright (C) {year}  {name of author}
+    Netdata
+    Copyright (C) 2020  Netdata, Inc.
```

Comment:
it is `Netdata, Inc.` since 2018 (i checked nd dashboard)

Reply:
ah cool, thanks :+1: